### PR TITLE
Improving shutdown time and changing SslContext creation.

### PR DESCRIPTION
**Pull request by sduskis**
The netty EventLoopGroup takes about 2 seconds to terminate.  It also shuts down when the JVM shuts down.  Older versions didn't wait for it to terminate, so this version shouldn't either.

Using GrpcSslContexts.forClient() as per Nathan's instruction instead of SslContext.newClientContext().

### CODE REVIEW

File: bigtable-grpc-interface/src/main/java/com/google/cloud/bigtable/grpc/BigtableSession.java [hunk 1]
```diff
@@ -99,28 +101,15 @@
...
-        // Each one needs its own SslContext.
-        return SslContext.newClientContext();
-      } catch (SSLException e) {
-        throw new IllegalStateException("Could not create an ssl context.", e);
-      }
+  static {
+
+    if (System.getProperty("java.version").startsWith("1.7.")) {
+      // The grpc cyphers only work in JDK 1.8+.  Use the default system cyphers for JDK 1.7.
+      sslBuilder.ciphers(null);
```
Comment (nmittler) [inline bigtable-grpc-interface/src/main/java/com/google/cloud/bigtable/grpc/BigtableSession.java]: You may also want to log a warning that they should use Java 8 to get more secure ciphers.

File: bigtable-grpc-interface/src/main/java/com/google/cloud/bigtable/grpc/BigtableSession.java [hunk 2]
```diff
@@ -99,28 +101,15 @@
...
-    @Override
-    public SslContext create() {
-      try {
-        // We create multiple channels via refreshing and pooling channel implementation.
-        // Each one needs its own SslContext.
-        return SslContext.newClientContext();
-      } catch (SSLException e) {
-        throw new IllegalStateException("Could not create an ssl context.", e);
-      }
+  static {
```
Comment (kevinsi4508) [inline bigtable-grpc-interface/src/main/java/com/google/cloud/bigtable/grpc/BigtableSession.java]: nit:
Can we make it a private static method?

private final static SslContextBuilder sslBuilder = getSslContextBuilder();

private static SslContextBuilder getSslContextBuilder() {
  SslContextBuilder sslBuilder = GrpcSslContexts.forClient();
  ...
  return sslBuilder;
}

File: bigtable-grpc-interface/src/main/java/com/google/cloud/bigtable/grpc/BigtableSession.java (hunk 1 above)
Comment (kevinsi4508) [inline bigtable-grpc-interface/src/main/java/com/google/cloud/bigtable/grpc/BigtableSession.java]: +1

**Comment by kevinsi4508** [comment]
LGTM

**Pull request merged**